### PR TITLE
Turns contents and fileLength variables into constants.

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

File: performance/PerformanceExampleSwift/ViewController.swift [hunk 1]
```diff
@@ -34,15 +34,16 @@ class ViewController: UIViewController {
     // Start tracing
     let trace = Performance.startTrace(name: "request_trace")
 
-    var fileLength = 0
-    var contents = ""
+	let contents: String
```
Comment (morganchen12) [inline performance/PerformanceExampleSwift/ViewController.swift]: Please fix the indents.

Comment (valeriyvan) [reply performance/PerformanceExampleSwift/ViewController.swift]: Hope, I have fixed it.

**Comment by valeriyvan** [comment]
Fixed

**Comment by morganchen12** [comment]
Thanks!

**Pull request merged**